Sketch.java

```diff
@@ -2,35 +2,50 @@
 
 public class Sketch extends PApplet {
 	
-	
   /**
-   * Called once at the beginning of execution, put your size all in this method
+   * This program draws a house
+    Author: Ethan Rodrigues
    */
   public void settings() {
-	// put your size call here
     size(400, 400);
   }
 
-  /** 
-   * Called once at the beginning of execution.  Add initial set up
-   * values here i.e background, stroke, fill etc.
-   */
   public void setup() {
     background(210, 255, 173);
   }
 
-  /**
-   * Called repeatedly, anything drawn to the screen goes here
-   */
   public void draw() {
-	  
-	// sample code, delete this stuff
-    stroke(128);
-    line(150, 25, 270, 350);  
 
-    stroke(255);
-    line(50, 125, 70, 50);  
-  }
+      // code for the grass
+        fill(0, 176, 26);
+        rect(0, 380, 400, 400);
+
+      // code for the house
+        fill(222, 139, 75);
+        rect(20, 120, 360, 400);
+
+      // code for the chimney
+        rect(60, 20, 50, 80);
+
+      // code for the roof
+        fill(125, 60, 10);
+        triangle(20, 120, 200, 15, 380, 120);
+
+      // code for the door and the doorknob
+        fill(237, 206, 183);
+        rect(50, 300, 80, 400);
+        ellipse(110, 370, 10, 10);
+
+      // code for the garage door
+        fill(120, 106, 99);
+        rect(220, 280, 120, 320);
+        fill(133, 107, 72);
+        rect(225, 285, 110, 320);
+      
+      // code for the windows
+        fill(192, 206, 207);
+        ellipse(100, 200, 80, 80);
```

Comment:
Align your comments with the code they refer to. Comments should not be outdented beyond corresponding code